src/view/com/feeds/FeedSourceCard.tsx

```diff
@@ -23,7 +22,7 @@ import {
 } from '#/state/queries/preferences'
 import {useFeedSourceInfoQuery} from '#/state/queries/feed'
 
-export const NewFeedSourceCard = observer(function FeedSourceCardImpl({
+export const FeedSourceCard = observer(function FeedSourceCardImpl({
```

Comment:
Had built a new version alongside the old. Replaced last old version with new and renamed.